app/models/user.rb

```diff
@@ -82,12 +82,14 @@ def secure_password
   # that is less than 8 hours old,
   # AND they would otherwise be in the call list
```

Comment:
Should this comment be updated to mention the new call line criteria, or is that implicit in the "would otherwise be in the call list" ?